src/config.js

```diff
@@ -521,6 +524,75 @@ export default class Config {
     }
   }
 
+  async findManifest(dir: string, isRoot: boolean): Promise<?Manifest> {
+    for (const registry of registryNames) {
+      const manifest = await this.tryManifest(dir, registry, isRoot);
+
+      if (manifest) {
+        return manifest;
+      }
+    }
+
+    return null;
+  }
+
+  async findWorktree(initial: string): Promise<?string> {
+    let previous = null;
```

Comment:
Ping about the question above.

I think it is related to my previous comment regarding `config.worktreeFolder || config.cwd`. 
Should resolving workspaces and root folders be hidden behind the configuration flag and whether workspaces is enabled in the root manifest?